src/rf/js/src/home/views.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+var React = require('react'),
+    asset = require('../core/utils').asset;
+
+var LoginView = React.createBackboneClass({
+    render: function() {
+        return <div>
```

Comment:
I started using that style for newer components. The only difference is you need to wrap in parens like this:

```
return (
    <div>
        <tags and tags>
    </div>
);
```

I'll update this.